discovery/create-and-anchor-did-web.mjs

```diff
@@ -0,0 +1,52 @@
+
+import { anchor, DID, generateKeyPair } from '@decentralized-identity/ion-tools';
+// import { anchor, DID, generateKeyPair } from './dist/esm/index.js';
+
+import { writeFile } from 'fs/promises';
+
+// Generate keys and ION DID
+let authnKeys = await generateKeyPair();
+let did = new DID({
+  content: {
+    publicKeys: [
+      {
+        id: 'key-1',
+        type: 'EcdsaSecp256k1VerificationKey2019',
+        publicKeyJwk: authnKeys.publicJwk,
+        purposes: [ 'authentication' ]
+      }
+    ],
+    services: [
+      {
+        id:'pfi',
+        'type': 'PFI',
+        'serviceEndpoint': 'did:web:pfi.mic4.com',
```

Comment:
if I understand correctly this is creating an ION DID with a service endpoint pointing to a did:web DID?

I would recommend altering this to having a new verification relationship linking the two DIDs, where the did:web lists the ION DID as its controller